New actions to bump homebrew and auto format action (#81)
cloudposse/atmos: README.md

For example, maybe you have your own existing CLI tools (e.g. using `terragrunt`). In this case, you may want to start by developing your own CLI.

## Install

### OSX

From Homebrew directly

```
brew install atmos
```

### cloudposse/packages

> Using container

Reviewer: don't think we need to limit it to containers only.
apk/apt will work in container and local hosts

> Using container

Reviewer: ```suggestion

```

> ```

Reviewer: remove 

```
ARG ATMOS_VERSION
```

ARG ATMOS_VERSION
```

> If using debian, then `RUN` this

Reviewer: ```suggestion
On Debian, run:
```

```

> apt-get install atmos@="$(ATMOS_VERSION}-*"

Reviewer: ```suggestion
apt install atmos@="<ATMOS_VERSION>-*"
```

```

> If using alpine, then `RUN` this

Reviewer: ```suggestion
On Alpine, run:
```

```

> apk add atmos@cloudposse~=$(ATMOS_VERSION}

Reviewer: ```suggestion
apk add atmos@cloudposse~=<ATMOS_VERSION>
```

```

### Go

> Get the latest version

Reviewer: ```suggestion
Install the latest version
```

```
go install github.com/cloudposse/atmos
```

Get a specific version

> NOTE: Since the version is passed in via `-ldflags`, the specific version will return `0.0.1` when running `atmos version`.

Reviewer: ```suggestion
__NOTE:__ Since the version is passed in via `-ldflags` during build, when running `go install` without using `-ldflags`, the CLI will return `0.0.1` when running `atmos version`.
```
